### ISSUE TITLE
How should I structure the Anti-Aesthetic essay?

### ISSUE THREAD

**Issue by devinhalladay** [GD399-OSD/another-green-world#2]
I'm currently thinking maybe I need a couple parts. 

1. What is an aesthetic? Why do aesthetics matter as a philosophical concept? Why are aesthetics important in design?
2. Why is the current view of aesthetics bad? What is the real etiology of an aesthetic? What special powers do aesthetics have over individuals and over culture?

I'm not sure if I need other parts to this essay. Tangential but potentially valuable things may include:
- What is the crisis of visual culture I keep ranting about on Twitter?
- Why is visual culture in a state of crisis?
- How do our aesthetic sensibilities impact politics and psychology at scale (i.e. on a social macro-level)?
- How do certain aesthetics contribute to the economy & to capitalism?

**Comment by bjornmeansbear** [comment]
Yes, describing “what is an aesthetic” seems like a good way to get started. Especially if you want to also define “anti-aesthetic” in contrast to that.
